source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -12,173 +12,421 @@ feature_status: Design
 
 ## Summary
 
-Incremental backup allows a user to backup only the changed blocks in
-all the disks belonging to a VM snapshot. The typical use case is backing
-up the current changes in the active snapshot. Another use case may be
-backing up the previous snapshot if a user added a snapshot since the
-last backup.
+Incremental backup allows a user to backup only the changed blocks since
+the last backup, instead of performing a full backup on every backup.
+If the VM is running during the backup, new data written after the
+backup is started is not included in the backup.
```

Comment:
Need to add some background on previous ways to backup, to understand what will be added.

This add 2 new important capabilities: 
- User can do now full backup without creating a snapshot for both raw and qcow2 disks
- User can do now incremental backup without creating snapshots for qcow2 disks